spec/Classes.tex

```diff
@@ -1566,8 +1566,8 @@ \subsection{Class Assignment}
 \end{chapelpre}
 \begin{chapel}
 // assume that C is a class
-var a = new owned C();
-var b:owned C; // default initialized to store `nil`
+var a:owned C? = new owned C();
```

Comment:
I would expect it to continue to have `var a = new owned C();`. Does it not work that way today?

Reply:
`a` is emptied two lines down then accessed after being emptied.